README.md

```diff
@@ -8,14 +8,13 @@ When the application is launched in a deployed environment we make no assumption
 This is mentioned here as a handy guide to pointing local applications to other services, and a reminder that developers adding new requirements would be advised to update the devOps team!
 
 Required environment parameters:
-- SERVER (mongo db url)
-- MONGO_USER
-- MONGO_PWD
-- DBNAME
-- GETH_NODE_URL  - url of a geth node for the app to talk to
-- GETH_NODE_PORT - port to open a websocket on
-- CHECKSUM_KEY ???
-- HASH_PREFIX ???
+- BCX_MONGO_SERVER_ADDRESS (mongo db url)
+- BCX_MONGO_USER
+- BCX_MONGO_PASSWORD
+- BCX_MONGO_DATABASE_NAME
+- BCX_GETH_NODE_URL  - url of a geth node for the app to talk to
+- BCX_GETH_NODE_PORT - port to open a websocket on
+- BCX_CHECKSUM_KEY ???
```

Comment:
@t-j-turner has pointed out that this README which they are following contains variables that are missing. Could you please split this out by service (such as housekeeper, deferred etc) and what environment variables are required for each? Thanks!